transcom/mymove
# B 20204 int

## [B-20204](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20204)

## Summary

This story tackles a defect where a value being returned as cents was being formatted using a dollar->cents formatter. This task simply removes this formatter to show the correct value in cents to the tenth of a cent.

The value in question is the baseline rate difference, which is calculated by subtracting the calculated EIA diesel value from the baseline GHC Diesel fuel price (static $2.5 value).

See the screenshot section below for an example of a correctly calculated and formatted value, noting that the difference between $2.814 and $2.500 dollars is 31.4 cents.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

To avoid having to set up a great amount of test data for a relatively small change, follow the instructions below for local dev testing:
1. Access the office app as a TIO
2. Locate the payment request with move code ORGSIT
3. Click Review Weight first to resolve the weight being 2000 lbs and over the 110%, move it down to 1500 lbs then approve.
4. Then, you should be able to view the service items and approve until you reach fuel surcharge, or alternatively go back to the move and click "Show request details" under the payment request. Then, locate fuel surcharge, and expand the calculations.
5. Ensure that the baseline rate difference is formatted to display in cents, to the tenth of a cent. Ensure that the value is the result of $2.50 minus the calculated EIA Diesel value ($2.814 in the example below)

### Frontend


## Screenshots
<img width="917" alt="image" src="https://github.com/transcom/mymove/assets/148906585/45c86577-85a9-4781-87c2-3d48d7912315">
